### PR TITLE
Fix for 3069

### LINKED ISSUE #3069: WIRED_MAC_AUTH should be "merged" with Ethernet-NoEAP
Title says it all.

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Ethernet-NoEAP and no WIRED-MAC-AUTH anymore

# Impacts
No

# Issue
fixes #3069

# Delete branch after merge
YES

# NEWS file entries
## Bug Fixes
* WIRED_MAC_AUTH and Ethernet-NoEAP merged (#3069)

### CODE REVIEW

**Comment by extrafu** [comment]
Should we offer backward compatibility? Ie., we keep supporting WIRED-MAC-AUTH but spew a message in the logs about this.

**Comment by julsemaan** [comment]
I believe we would need to have an SQL statement in the upgrade script to rename the connection type in the locationlog

**Comment by jrouzierinverse** [comment]
Would the profile filters need to be changed also?

**Comment by julsemaan** [comment]
@jrouzierinverse 

Yes good point, there needs to be a script for that too

**Review by julsemaan — COMMENTED** [review]
@fdurand 

Few things to check out based on @jrouzierinverse comments and mine

**Comment by fdurand** [comment]
BUMP

File: UPGRADE.asciidoc [hunk 1]
```diff
@@ -1755,3 +1755,14 @@ Queue Stats maintenance job removal
 The queue_stats maintenance job has been deprecated in favor of using pfstats. In order to remove configuration related to this maintenance job, run:
 
   /usr/local/pf/addons/upgrade/to-8.2-pfmon-conf.pl
+
+Update connection_type from WIRED_MAC_AUTH to Ethernet-NoEAP
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We merged the WIRED_MAC_AUTH and Ethernet-NoEAP to Ethernet-NoEAP so the configuration needs to be updated, to do that run:
+
+ sed -i "s/WIRED_MAC_AUTH/Ethernet-NoEAP/g" conf/profiles.conf
```
Comment (julsemaan) [inline UPGRADE.asciidoc]: Use absolute paths

**Comment by fdurand** [comment]
BUMP

File: UPGRADE.asciidoc [hunk 2]
```diff
@@ -1755,3 +1755,14 @@ Queue Stats maintenance job removal
...
+
+Update connection_type from WIRED_MAC_AUTH to Ethernet-NoEAP
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We merged the WIRED_MAC_AUTH and Ethernet-NoEAP to Ethernet-NoEAP so the configuration needs to be updated, to do that run:
+
+ sed -i "s/WIRED_MAC_AUTH/Ethernet-NoEAP/g" /usr/local/pf/conf/profiles.conf
+ sed -i "s/WIRED_MAC_AUTH/Ethernet-NoEAP/g" /usr/local/pf/conf/vlan_filters.conf
+ sed -i "s/WIRED_MAC_AUTH/Ethernet-NoEAP/g" /usr/local/pf/conf/radius_filters.conf
+ sed -i "s/WIRED_MAC_AUTH/Ethernet-NoEAP/g" /usr/local/pf/conf/switch_filters.conf
+ sed -i "s/WIRED_MAC_AUTH/Ethernet-NoEAP/g" /usr/local/pf/conf/authentication.conf
```
Comment (jrouzierinverse) [inline UPGRADE.asciidoc]: To make it easier and reduce mistakes can you put all those into a single shell script.

**Comment by jrouzierinverse** [comment]
rebase

**Comment by fdurand** [comment]
BUMP

File: UPGRADE.asciidoc [hunk 3]
```diff
@@ -1794,3 +1794,15 @@ To upgrade the database schema, run the following command:
 Once completed, update the file /usr/local/pf/conf/currently-at to match the new release number (PacketFence X.Y.Z).
 
 
+  /usr/local/pf/addons/upgrade/to-8.2-pfmon-conf.pl
```
Comment (julsemaan) [inline UPGRADE.asciidoc]: This doesn't belong in this PR

**Review by julsemaan — COMMENTED** [review]
There is also @jrouzierinverse's comment on the upgrade notes to see if we do or not

**Comment by fdurand** [comment]
BUMP
